tests/test_utils.py

```diff
@@ -94,3 +94,44 @@ def test_invalidTimespan(self):
             getMidpointFromTimespan(timespan_none_both, allowUnbounded=True)
         with self.assertRaises(ValueError):
             getMidpointFromTimespan(timespan_none_both, allowUnbounded=False)
+
+    def test_ssObjectID_to_objID_and_objID_to_ssObjectID(self):
+        """Convert between ssObjectIDs and MPC packed designations
+        """
+        allowed_strings = [ch for ch in 'ABCDEFGHIJKMNOPQRSTUVWXYZ0123456789 -'] \
+            + ['A' * i for i in range(2, 8)] + ['Z' * i for i in range(2, 8)] \
+            + ['J95X00A', 'J95X01L', 'J95F13B', 'J98SA8Q', 'J98SC7V', 'J98SG2S'] \
+            + ['K99AJ3Z', 'K08Aa0A', 'K07Tf8A', 'PLS2040', 'T1S3138', 'T2S1010', 'T3S4101']
+        allowed_flags = [i for i in range(0, 256)]
+        for allowed_string in allowed_strings:
+            for allowed_flag in allowed_flags:
+                self.assertEqual((allowed_string, allowed_flag),
+                                 ssObjectID_to_objID(objID_to_ssObjectID(allowed_string,
+                                                                         allowed_flag)))
+
+    def test_invalid_ssObjectID_to_objID_and_objID_to_ssObjectID(self):
+        """Convert between ssObjectIDs and MPC packed designations
+        """
+        allowed_strings = [ch for ch in 'ABCDEFGHIJKMNOPQRSTUVWXYZ0123456789 -'] \
+                + ['A' * i for i in range(2, 8)] + ['Z' * i for i in range(2, 8)] \
+                + ['J95X00A', 'J95X01L', 'J95F13B', 'J98SA8Q', 'J98SC7V', 'J98SG2S'] \
+                + ['K99AJ3Z', 'K08Aa0A', 'K07Tf8A', 'PLS2040', 'T1S3138', 'T2S1010', 'T3S4101']
+        allowed_flags = [i for i in range(0, 256)]
+        disallowed_flags = [-999999999, -512, -256, -255, -1, 256, 512, 99999999]
+        disallowed_strings = ['Ā', '🔭', 'A' * 8, ' ' * 8, 'A' * 128]
```

Comment:
I think putting an emoji in Python code might be going a little too far (though it's hilarious that GitHub can print it!)

Reply:
What about the empty string? That seems like a natural edge case.